twitter-eventlistener-plugin/src/main/java/com/twitter/presto/plugin/eventlistener/TwitterEventListener.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twitter.presto.plugin.eventlistener;
+
+import com.facebook.presto.spi.eventlistener.EventListener;
+import com.facebook.presto.spi.eventlistener.QueryCompletedEvent;
+import com.facebook.presto.spi.eventlistener.QueryCreatedEvent;
+import com.facebook.presto.spi.eventlistener.SplitCompletedEvent;
+
+import io.airlift.log.Logger;
+
+public class TwitterEventListener implements EventListener
+{
+  private static final Logger log = Logger.get(TwitterEventListener.class);
+  private final QueryCompletedEventScriber scriber = new QueryCompletedEventScriber();
+
+  @Override
+  public void queryCreated(QueryCreatedEvent queryCreatedEvent)
+  {
+  }
+
+  @Override
+  public void queryCompleted(QueryCompletedEvent queryCompletedEvent)
+  {
+    log.info(Stringify.toString(queryCompletedEvent));
```

Comment:
👍 to get rid of the detailed logging. Maybe we instead log a few key fields that might be useful?